### DISCUSSION TITLE
[AWS] Meilisearch arm64 AMI

### DISCUSSION THREAD

**Discussion by jyuno426** [meilisearch/product#726] [Feedback & Feature Proposal]
I think it would be great to support official Meilisearch ami with arm64 architecture.

Do you have any plan to support it?

**Comment by macraig** [comment]
Hi @jyuno426 , thanks for taking the time to open a discussion :) We don't currently plan to support it as creating a second image for arm64 would increase costs and require maintenance without any clear value added. That said, we're always open to revisiting our decisions if we discover there's enough appetite for the need. What value would it add to your use case? Is the current image limiting the way you use Meilisearch today?
